test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -529,10 +529,10 @@ void IHttpHeadersHandler.OnHeader(Span<byte> name, Span<byte> value)
             await StartStreamAsync(1, _browserRequestHeaders, endStream: false);
             await StartStreamAsync(3, _browserRequestHeaders, endStream: false);
 
-            await SendDataAsync(1, _helloBytes, endStream: false);
+            await SendDataAsync(1, _helloBytes, endStream: true);
             Assert.True(stream1Read.WaitOne(TimeSpan.FromSeconds(10)));
 
-            await SendDataAsync(3, _helloBytes, endStream: false);
+            await SendDataAsync(3, _helloBytes, endStream: true);
```

Comment:
Was this change necessary for the test to pass?

Reply:
Leftover from a prior version, reverted.